src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -130,8 +130,8 @@ public static int Main(string[] args)
 
             createRulesetAndEditorconfig(
                 "AllRulesEnabled",
-                "All Rules Enabled with default severity",
-                "All Rules are enabled with default severity. Rules with IsEnabledByDefault = false are force enabled with default severity.",
+                "All Rules Enabled as build warnings",
```

Comment:
Need a similar change for https://github.com/dotnet/roslyn-analyzers/blob/33582d0369adb7657731b611f4811e981dcb2f7f/src/Tools/GenerateDocumentationAndConfigFiles/Program.cs#L152-L157